src/v1.js

```diff
@@ -233,6 +233,7 @@ api.declare({
 
   // Try to create a Hook entity
   try {
+
```

Comment:
Careful about adding unnecessary empty lines like this.  They don't hurt anything but they take up space on our screens! :)

Reply:
it was removed. :)